src/components/landing/ScrollControl.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
+import { useEffect } from 'react';
+interface ScrollContainerProps {
+  slideNumber: number;
+  setSlideNumber: (slideNumber: number) => void;
+}
+
+function ScrollContainer(props: ScrollContainerProps) {
+  const { slideNumber, setSlideNumber } = props;
+
+  useEffect(() => {
+    console.log(slideNumber);
+  }, [slideNumber]);
+
+  return (
+    <StScrollContainer>
+      <li className={slideNumber === 1 ? 'isActive' : 'unActive'} onClick={() => setSlideNumber(1)}></li>
+      <li className={slideNumber === 2 ? 'isActive' : 'unActive'} onClick={() => setSlideNumber(2)}></li>
+      <li className={slideNumber === 3 ? 'isActive' : 'unActive'} onClick={() => setSlideNumber(3)}></li>
+      <li className={slideNumber === 4 ? 'isActive' : 'unActive'} onClick={() => setSlideNumber(4)}></li>
+      <li className={slideNumber === 5 ? 'isActive' : 'unActive'} onClick={() => setSlideNumber(5)}></li>
+      <li className={slideNumber === 6 ? 'isActive' : 'unActive'} onClick={() => setSlideNumber(6)}></li>
+    </StScrollContainer>
+  );
+}
```

Comment:
중복되는 코드인만큼 map함수로 바꿔볼라고,, 했는데 왠지 딱히 줄어진 거 같지 않은데 기분탓일까여 좋은 아이디어 있으신 분ㅎㅎ

```suggestion
  const slideNumbersArr = [1, 2, 3, 4, 5];
  const slideNumbersList = slideNumbersArr.map((number) => (
    <li
      key={number}
      className={slideNumber === number ? 'isActive' : 'unActive'}
      onClick={() => setSlideNumber(number)}>
      {number}
    </li>
  ));

  return <StScrollContainer>{slideNumbersList}</StScrollContainer>;
}
```

Reply:
나 이런거 진짜 좋아해....사랑해요

Reply:
```suggestion
  const slideNumbersArr = [1, 2, 3, 4, 5, 6];
  return (
    <StScrollContainer>
      {slideNumbersArr.map((number) => {
        return (
          <li
            key={number}
            className={slideNumber === number ? 'isActive' : 'unActive'}
            onClick={() => setSlideNumber(number)}></li>
        );
      })}
    </StScrollContainer>
  );
}
```

요렇게 바꿨서요

Reply:
slideNumbersArr => slideNumberList로 바꿉시당 (컨벤션 참고 !!) 😉

Reply:
"배열과 관련된 변수명은 __LIst" 나 오늘 자기 전에 코드 컨벤션 정독 한번더 할 것이에요 ;;